src/ui/buttons/button.jsx

```diff
@@ -215,7 +215,7 @@ export function Button({ fundingSource, style, multiple, locale, env, fundingEli
             ].join(' ') }
         >
             <div
-                role='button'
+                role='link'
```

Comment:
@paypal/checkout-sdk please review and share any concerns. This change should improve screen reader behavior.

Reply:
For more info on the reference for the discussion with accessibility team here is the reference - https://paypal.slack.com/archives/C1S7705Q8/p1648757897237639?thread_ts=1648751542.872819&cid=C1S7705Q8